db/migration/1731360326761-CloudflareImages.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class CloudflareImages1731360326761 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            ALTER TABLE images 
+                ADD COLUMN cloudflareId VARCHAR(255) NULL,
+                MODIFY COLUMN googleId VARCHAR(255) NULL
+        `)
```

Comment:
I believe we want a uniqueness constraint on `cloudflareId`, i.e. a cloudflare image should only have one filename in our system.